### PR TITLE
Wpi staging comp merge ready

### PR DESCRIPTION
**Pull request by noahheller**

### CODE REVIEW

File: src/main/java/frc/robot/commands/arm/InitialMoveArm.java [hunk 1]
```diff
@@ -73,6 +87,7 @@ public void end(boolean Interrupted) {
 
     @Override
     public boolean isFinished() {
+        SmartShuffleboard.put("Arm","InitalMoveArmFinish", arm.getAnalogValue());
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/arm/InitialMoveArm.java]: Move this to "end", and convert to log a string message

File: src/main/java/frc/robot/commands/arm/ManualMoveArm.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ public void execute() {
     public void end(boolean Interrupted) {
         super.end(Interrupted);
         arm.setVoltage(0.0);
-        new HoldArmPID(arm, arm.getAnalogValue()).schedule();
+        new HoldArmPID(arm, arm.getAnalogValue()+2).schedule();
```
Comment (bengold22) [inline src/main/java/frc/robot/commands/arm/ManualMoveArm.java]: Remove the +2

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 3]
```diff
@@ -91,11 +94,13 @@ public void periodic() {
     }
     SmartShuffleboard.put("Arm", "Encoder angle",getAnalogValue());
     SmartShuffleboard.put("Arm", "Encoder",analogSensor.getPosition());
+    SmartShuffleboard.put("Arm","ArmTarget",pidreference);
```
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/Arm.java]: Move all these values (Encoder angle, Encoder, ArmTarget) into debug section

**Review by bengold22 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/arm/InitialMoveArm.java (hunk 1 above)
Comment (noahheller) [reply src/main/java/frc/robot/commands/arm/InitialMoveArm.java]: done

File: src/main/java/frc/robot/commands/arm/ManualMoveArm.java (hunk 2 above)
Comment (noahheller) [reply src/main/java/frc/robot/commands/arm/ManualMoveArm.java]: done

File: src/main/java/frc/robot/subsystems/Arm.java (hunk 3 above)
Comment (noahheller) [reply src/main/java/frc/robot/subsystems/Arm.java]: done

**Review by bengold22 — APPROVED** [review]

**Pull request merged**